src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -75,15 +75,17 @@
 
 
 (defn render-block
-  [blocks {:keys [type literal children]} chat-id]
+  [blocks {:keys [type literal children]} chat-id style-override]
   (case (keyword type)
     :paragraph
     (conj blocks
           [rn/view {:style style/paragraph}
            (reduce
             (fn [acc e]
               (render-inline acc e chat-id))
-            [quo/text]
+            [quo/text
+             {:style {:color (when (seq style-override)
```

Comment:
(sorry for comment jumping). This here is my main concern as it couples the text color changing to when this `style-override` prop is used which is a bit unexpected.

One alternative is to just pass the `container-style` and `text-style` as 
props to `render-parsed-text` function like this:

```
(defn render-parsed-text
  [{:keys [content chat-id edited-at container-style text-style]} ]
  ^{:key (:parsed-text content)}
  [rn/view {:style container-style}
   (reduce (fn [acc e]
             (render-block acc e chat-id text-style))
           [:<>]
           (cond-> (:parsed-text content)
             edited-at
             add-edited-tag))])

(defn text-content
  [message-data]
  [rn/view
   [render-parsed-text (assoc message-data :container-style style/parsed-text-block)]
   [link-preview/view message-data]])
   ```
   
   and so in your file it would be:
   ```
   [message-view/render-parsed-text
      {:content        content
       :chat-id        chat-id
       :container-style/text-style
       :text-style {:color colors/white}]
    ```
    
   The only thing then that is still a bit off to me is why this text-style is also not applied to the other cases in the `render-text-block` function, i.e:   `:edited-block`,  `:blockquote`,  `:codeblock`. However maybe that's fine for now. 
   
   Anyway I'll leave it to you for how best to approach it as you know this code better :)

Reply:
You're totally right, I considered the other types of textual message in the process.

Reply:
Fixed